Command/Validators.php

```diff
@@ -127,6 +127,19 @@ public static function validateEntityName($entity)
         return $entity;
     }
 
+    public static function validateEntityPropertyName($property)
+    {
+        if (in_array($property, self::getReservedWords())) {
+            throw new \InvalidArgumentException(sprintf('Name "%s" is a PHP reserved word.', $property));
+        }
+
+        if (!preg_match('/^[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*$/', $property)) {
```

Comment:
if you use the pattern directly, you can remove the method from the generator class